### DISCUSSION TITLE
I did aply for github copilot i got approved and i didnt get a copilot and other benefits

### DISCUSSION THREAD

**Discussion by Klocu-jpg** [community/community#53876] [GitHub Education]
### Hi everyone,

So ill drop u a  few screnshots what i was exacly doing.

so at first i did a request for a mainly github copilot and i did get approved but i did not get access to copilot idk why.

![image](https://user-images.githubusercontent.com/129229751/234884074-b19d8684-0c9a-4820-84f4-ac466a5daa89.png)

and i cant turn it on!!! 

![image](https://user-images.githubusercontent.com/129229751/234884537-d7be905b-40cd-4f75-ac6e-d19abb68baf8.png)

what can i do pls help!!

**Comment by bbkx226** [comment]
Hi @Klocu-jpg, have you tried login GitHub Copilot using your verified GitHub account?

**Reply by Klocu-jpg** [reply]
wdym veryfied account i veryfied email and i think its all i did verified. Can u tell my how can u do it>?

**Reply by Klocu-jpg** [reply]
Yeah i do have veryfied account!
still same results

**Reply by bbkx226** [reply]
For me, I used GitHub Copilot Extension in VS Code, and it requires me to log in using my GitHub account. Then, I got the access and am able to use it

**Reply by Klocu-jpg** [reply]
i already did it and i did use it in inteliJ too and it did not work!

**Comment by Klocu-jpg** [comment]
![Zrzut ekranu 2023-05-13 121413](https://github.com/community/community/assets/129229751/2d60495b-e43d-4af4-a2f8-1e472e2debbe)

**Reply by bbkx226** [reply]
Did you set your global username and email on your local machine?

**Reply by Klocu-jpg** [reply]
yes

**Comment by Ntshangase** [comment]
Hey @Klocu-jpg 

To make things easier you need to make sure your IDE is linked to your GitHub account.

**Reply by Klocu-jpg** [reply]
I do have connected IDE to github
